README.rst

```diff
@@ -25,35 +25,43 @@
 Introduction to libEnsemble
 ===========================
 
-libEnsemble is a Python library to coordinate the concurrent evaluation of
-dynamic ensembles of calculations. The library is developed to use massively
-parallel resources to accelerate the solution of design, decision, and
-inference problems and to expand the class of problems that can benefit from
-increased concurrency levels.
-
-libEnsemble aims for the following:
-
-• Extreme scaling
-• Resilience/fault tolerance
-• Monitoring/killing of tasks (and recovering resources)
-• Portability and flexibility
-• Exploitation of persistent data/control flow
-
-The user selects or supplies a *generator function* that produces
-input parameters for a *simulator function* that performs and monitors
-simulations. For example, the generator function may contain an
-optimization routine to generate new simulation parameters on-the-fly based on
-the results of previous simulations. Examples and templates of such functions are
-included in the library.
-
-libEnsemble employs a manager/worker scheme that can run on various
-communication media (including MPI, multiprocessing, and TCP); interfacing with
-user-provided executables is also supported. Each worker can
-control and monitor any level of work, from small subnode tasks to huge
-many-node simulations. An executor interface is provided to ensure that scripts
-are portable, resilient, and flexible; it also enables automatic detection of
-the nodes and cores available to the user, and can dynamically assign resources
-to workers.
+libEnsemble is a Python_ toolkit for coordinating asynchronous and dynamic ensembles of calculations.
+
+libEnsemble workflows can run on and across laptops, leadership-class machines and facilities,
```

Comment:
Yeah, I think the across is not that clear. I think I would say
"libEnsemble workflows can run on laptops, clusters, and leadership-class machines,"

or to generalize a bit more could say
"libEnsemble workflows can run on diverse platforms including laptops, clusters, and leadership-class machines,"

Reply:
If you want to keep 'across', thats okay.

Reply:
libEnsemble workflows can run on or across  laptops, clusters, and leadership-class machines